### DISCUSSION TITLE
Exporting a centroid-only model?

### DISCUSSION THREAD

**Discussion by jkbhagatio** [talmolab/sleap#1541] [General]
I believe that currently there is no way to export a centroid-only model via the SLEAP API or CLI. E.g., trying to using `sleap-export` yields the following error: 
```
ValueError: Ground truth data was not detected... Please load both models when predicting on non-ground-truth data.
```

There are cases where we find it useful to have a centroid-only model, and @bruno-f-cruz has this workaround for using keras directly to perform this export: https://github.com/bruno-f-cruz/Sleap_export-Bonsai/blob/main/sleap_scripts/export_networks.ipynb (see the cell that begins with the comment "Export a centroid model") 

It would be nice to have the ability to do this directly via `sleap-export`. Is there a reason this isn't supported? If not, maybe this Discussion can be converted into an Issue?

**Comment by gitttt-1234** [comment]
Closing this discussion here, this feature request would be handled in [`sleap-nn`](https://github.com/talmolab/sleap-nn/issues/508), as SLEAP now uses sleap-nn for all neural-network workflows.
